test/unit/unit_tests_err.cpp

```diff
@@ -0,0 +1,73 @@
+
+#include <limits.h>
+#include "gtest/gtest.h"
+
+
+#ifdef __cplusplus
```

Comment:
It's always c++, isn't it (c++ code follows)?

Reply:
yes. removing it